venue.md

```diff
@@ -2,26 +2,12 @@
 title: Venue
 ---
 
-LambdAle will be held in the ballroom at [Balham Bowls Club](http://balhambowlsclub.com/).
+LambdAle will be held in the ballroom at [The George IV Pub](https://www.georgeiv.co.uk).
 
-The address is [7-9 Ramsden Rd, London SW12 8QX](https://www.google.com/maps/place/Balham+Bowls+Club/@51.444971,-0.152492,16z/data=!4m5!3m4!1s0x0:0x4928317423696e4d!8m2!3d51.4449078!4d-0.1525026?hl=en-GB).
+The address is [185 High Road, Chiswick, London, W4 2DR](https://goo.gl/maps/2kHAdvHpHqJ2).
 
-## Map
-
-<iframe src="https://www.google.com/maps/embed?pb=!1m14!1m8!1m3!1d19893.772141396432!2d-0.152503!3d51.444908!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x0%3A0x4928317423696e4d!2sBalham+Bowls+Club!5e0!3m2!1sen!2suk!4v1518043181690" width="300" height="300" frameborder="0" style="border:0" allowfullscreen></iframe>
+<iframe src="https://www.google.com/maps/embed?pb=!4v1550427124320!6m8!1m7!1sCAoSLEFGMVFpcE8yekNEajNfWlJDUjM5OXV4OXZrMWpjX2R0dkdUQ00zekd5cXI0!2m2!1d51.49242125220601!2d-0.2570486842103037!3f168.99!4f8.120000000000005!5f0.4000000000000002" width="600" height="450" frameborder="0" style="border:0" allowfullscreen></iframe>
 
 ## Directions
 
-From central London, either take the Northern line southbound or take National Rail from London Victoria towards Sutton (usually platform 1 or 2).
-
-Balham Bowls Club is only a couple of minutes' walk from Balham station.
-
-From Balham station, walk north up Balham High Street. Take the first left onto Ramsden Road. The pub is on your left.
-
-<a title="By Ewan Munro from London, UK [CC BY-SA 2.0 (https://creativecommons.org/licenses/by-sa/2.0)], via Wikimedia Commons" href="https://commons.wikimedia.org/wiki/File%3ABalham_Bowls_Club%2C_Balham%2C_SW12_(7047537451).jpg"><img width="512" alt="Balham Bowls Club, Balham, SW12 (7047537451)" src="https://upload.wikimedia.org/wikipedia/commons/thumb/a/ad/Balham_Bowls_Club%2C_Balham%2C_SW12_%287047537451%29.jpg/512px-Balham_Bowls_Club%2C_Balham%2C_SW12_%287047537451%29.jpg"/></a>
-
-Somebody will be at the door to greet you. The conference is in the ballroom upstairs.
-
 ## Accessibility
```

Comment:
The venue is on the ground floor and has wheelchair access.

If you have a disability and you would like to discuss your requirements, please get in touch at [info@lambdale.org](mailto:info@lambdale.org). We will do our best to ensure anybody who wants to attend LambdAle can do so.